deeppavlov/configs/ner/ner_kb_rus.json

```diff
@@ -1,7 +1,7 @@
 {
   "dataset_reader": {
     "class_name": "sq_reader",
-    "data_path": "/home/dmitry/download/ner_ontonotes"
+    "data_path": "{DOWNLOADS_PATH}/ner_sq/SQ_rus_dataset_zs.pckl"
```

Comment:
Are you sure that you don't need to change any code in the `sq_reader` after changing `data_path` format?

Reply:
Where `sq_reader` can be found? GitHub search gives nothing: https://github.com/deepmipt/DeepPavlov/search?q=sq_reader+path:/deeppavlov/dataset_readers&unscoped_q=sq_reader+path:/deeppavlov/dataset_readers

Reply:
Thank you, @StrikerRUS for noticing! It's in this pull request now.

Reply:
Thanks for fixing this!